addons/website/views/snippets/s_company_team_basic.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" name="Team">
+    <section class="s_company_team o_colored_level o_cc o_cc1 pt64 pb64" data-snippet="s_company_team_basic" data-name="Team">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="10" style="column-gap: 32px;">
+                <div class="o_grid_item g-col-lg-12 g-height-2 col-lg-12" style="grid-area: 1 / 1 / 3 / 13; --grid-item-padding-y: 32px; --grid-item-padding-x: 0px; z-index: 5;">
+                    <h3 class="o_default_snippet_text" style="text-align: center;">Meet our experts</h3>
+                </div>
+                <div class="o_grid_item g-col-md-6 g-col-lg-3 g-height-8 col-md-6 col-lg-3" data-name="Team Member" style="grid-area: 3 / 1 / 11 / 4; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px; z-index: 1; text-align: center;">
+                    <div class="o_not_editable" contenteditable="false">
+                        <img alt="" class="o_editable_media img-fluid rounded-circle w-lg-100" src="/web/image/website.s_company_team_image_1"/>
+                    </div>
+                    <p>
+                        <br/>
+                    </p>
+                    <h4 class="h5-fs">Tony Fred, CEO</h4>
+                    <p class="text-800">Chief Executive Officer</p>
+                </div>
+                <div class="o_grid_item g-col-md-6 g-col-lg-3 g-height-8 col-md-6 col-lg-3" data-name="Team Member" style="grid-area: 3 / 4 / 11 / 7; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px; z-index: 2; text-align: center;">
```

Comment:
I the user sets the inner text to `text-align: left;`, this rule will this remain here, doing nothing.
I propose to set `text-align: center;` to the inner text elements instead 😉
```suggestion
                <div class="o_grid_item g-col-md-6 g-col-lg-3 g-height-8 col-md-6 col-lg-3" data-name="Team Member" style="grid-area: 3 / 4 / 11 / 7; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px; z-index: 2;">
```